v0/destinations/am/utils.js

```diff
@@ -61,10 +61,19 @@ function getPlatform(payload, sourceKey) {
     : payloadVal;
 }
 
+// eslint-disable-next-line consistent-return
```

Comment:
Return `undefined` by default and remove this comment

Reply:
updated